content_handler/accessibility_bridge.cc

```diff
@@ -20,7 +20,7 @@ AccessibilityBridge::AccessibilityBridge(app::ApplicationContext* context)
     : writer_(context->ConnectToEnvironmentService<maxwell::ContextWriter>()) {}
 
 void AccessibilityBridge::UpdateSemantics(
-    const std::vector<blink::SemanticsNode>& update) {
+    const std::unordered_map<int32_t, blink::SemanticsNode>& update) {
```

Comment:
This cant possibly compile. The argument is being iterated over and the pair wont contain the `id` property.

Reply:
Maybe `typedef` this to something and document the key.

Reply:
Good catch. Which target uses this code?

Reply:
I guess it's the "content_handler". Can I build and test it on a Mac? I can't find anything about it in https://github.com/flutter/engine/blob/master/CONTRIBUTING.md.

Reply:
Checking that the default Fuchsia target builds should be fine. https://fuchsia.googlesource.com/docs/+/HEAD/getting_started.md

Reply:
This look good now, right?